### ISSUE TITLE
9.0.4 unable to correctly determine architecture on Mac

### ISSUE THREAD

**Issue by gpaciga** [webdriverio/selenium-standalone#833]
## Expected Behavior
Download URL on a Mac machine should resolve to either
https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/116.0.5845.96/mac-arm64/chromedriver-mac-arm64.zip
or
https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/116.0.5845.96/mac-x64/chromedriver-mac-x64.zip
depending on the architecture of the terminal.

Since 9.0.4, it is resolving to https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/116.0.5845.96//chromedriver-undefined.zip

In v9.0.3, this worked as expected.

## Current Behavior
Since 9.0.4, it is resolving to https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/116.0.5845.96//chromedriver-undefined.zip

## Steps to Reproduce

```
npx selenium-standalone install --singleDriverInstall=chrome
```

## Logs

```
11:13 $ npx selenium-standalone install --singleDriverInstall=chrome
----------
selenium-standalone installation starting
----------

---
selenium install:
from: https://github.com/SeleniumHQ/selenium/releases/download/selenium-4.9.0/selenium-server-4.9.0.jar
to: /Users/gpaciga/git/automation-testing/selenium-standalone/node_modules/selenium-standalone/.selenium/selenium-server/4.9.0/selenium-server.jar
---
chrome install:
from: https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/116.0.5845.96//chromedriver-undefined.zip
to: /Users/gpaciga/git/automation-testing/selenium-standalone/node_modules/selenium-standalone/.selenium/chromedriver/latest-mac-x64/chromedriver

Error in "getDownloadStream". It may be due to the specified edge driver version edgedl is unavailable for current platform. Try downloading a different version of edge driver
See more details below:
404 https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/116.0.5845.96//chromedriver-undefined.zip
Response code 404 (Not Found)
/Users/gpaciga/git/automation-testing/selenium-standalone/node_modules/selenium-standalone/lib/install.js:292
          throw new Error('Could not download ' + downloadUrl);
          ^

Error: Could not download https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/116.0.5845.96//chromedriver-undefined.zip
    at Request.<anonymous> (/Users/gpaciga/git/automation-testing/selenium-standalone/node_modules/selenium-standalone/lib/install.js:292:17)
    at Object.onceWrapper (node:events:628:26)
    at Request.emit (node:events:513:28)
    at emitErrorNT (node:internal/streams/destroy:157:8)
    at emitErrorCloseNT (node:internal/streams/destroy:122:3)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)
```

## Your Environment
`selenium-standalone` version 9.0.4
Tested on a Mac in both native and Rosetta terminals.

**Comment by gpaciga** [comment]
I'd be happy to contribute the unit tests from https://github.com/gpaciga/selenium-standalone/tree/chrome-mac-arch-argument-fix that I think would have avoided this, but I can't run tests at all for 9.0.4, so can't confirm if they're still valid.

**Comment by gpaciga** [comment]
confirmed that has been fixed in 9.0.5
